sirepo/pkcli/job_cmd.py

```diff
@@ -109,33 +111,42 @@ def _do_compute(msg, template):
 def _do_fastcgi(msg, template):
     import socket
 
+    def _recv():
+        r = b''
+        while True:
```

Comment:
This loop needs a stop before it runs out of memory. I think if we set 1e8 as the max message size, we should enforce it. That is, it should look something like:

```py
b  = s.recv(int(_MAX_MSG))
if not b:
    pkdlog('socket closed abruptly msg={}', msg)
    # maybe a different exception that can be bubbled up easily below
    raise RuntimeError('socket closed abruptly') 
if len(b) + len(r) > _MAX_MSG:
    raise RuntimeError('message larger than {} bytes',  _MAX_MSG)
r += b
if r[-1:] == b'\n':
    return pkjson.load_any(r)
```

Something like that.